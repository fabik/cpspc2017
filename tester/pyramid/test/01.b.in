15 50
482255 890 60 3 3 2 9 2 5 7 3 2 7 5 6
2586
450054
510858
366089
625674
882112
875638
298093
42886
728399
512993
428965
157315
731348
409672
277785
730131
125643
747810
366968
967371
60673
667564
181518
327859
505421
25738
677457
725445
322764
350665
163882
246238
509412
997292
914131
217560
800276
52372
320079
242911
117639
824053
627787
396172
362555
992428
342808
127578
238206
