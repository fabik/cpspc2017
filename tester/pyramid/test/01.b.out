3353
583404
662223
474560
811059
1143478
1135086
386416
55593
944222
664990
556066
203926
948044
531056
360092
946466
162870
969384
475700
1254000
78651
865360
235300
425002
655175
33364
878185
940390
418398
454565
212439
319197
660350
1292786
1184984
282023
1037395
67890
414916
314884
152495
1068216
813799
513556
469978
1286481
444381
165380
308786
